projects/kit/components/push/push.ts

```diff
@@ -0,0 +1,5 @@
+import {TuiPushComponent} from './push.component';
+import {TuiPushDirective} from './push.directive';
+import {TuiPushAlert} from './push-alert.component';
+
+export const TuiPush = [TuiPushComponent, TuiPushAlert, TuiPushDirective] as const;
```

Comment:
Update migration

https://github.com/taiga-family/taiga-ui/blob/d631e0a1b790c87ee5586b11e1ad61534c4a8cf3/projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts#L195-L201

Reply:
fixed